privacyidea/lib/applications/offline.py

```diff
@@ -171,7 +171,9 @@ def get_authentication_item(token_type,
             if token_type.lower() == "webauthn":
                 # return the pubkey and the credential_id (contained in the otpkey)
                 ret["response"] = {"pubkey": token_obj.get_tokeninfo("pubKey"),
-                                   "credential_id": token_obj.decrypt_otpkey()}
+                                   "credential_id": token_obj.decrypt_otpkey(),
+                                   "rpId": token_obj.get_tokeninfo("relying_party_id")}
```

Comment:
why not call it `rp_id` to be more consistent with `credential_id`?

Reply:
see my comment in the pr

Reply:
Ok, i didn't get the comment at first :-)